stylix/get-maintainers.nix

```diff
@@ -0,0 +1,17 @@
+{
+  pkgs,
+  lib,
+  ...
+}@args:
+let
+  ghIds = lib.mapAttrs' (name: value: {
+    "modules/${name}" = builtins.concatMap (m: m.github or [ ]) value.maintainers;
+  }) (import ./meta.nix args);
+in
+pkgs.stdenvNoCC.mkDerivation {
+  name = "get-maintainers";
+  src = ./.;
+  buildPhase = ''
+    echo ${builtins.toJSON ghIds} > $out/gh_ids.json
+  '';
+}
```

Comment:
This improve caching, this should not depend on `src = /stylix;` Either the `src` attribute is replaced with `dontUnpack = true;`, or an appropriate `runCommand` variant is used.

It might be more idiomatic to output simply to the `$out` file instead of treating `$out` as a directory.

Also, consider replacing `echo ${string} > $out` with the more reliable `cp ${file} $out` variant.

Reply:
This looks like it could just be a `writeText` or `writeTextDir`